core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-add-variable/dot-add-variable.component.ts

```diff
@@ -35,6 +35,6 @@ export class DotAddVariableComponent implements OnInit {
      * @memberof DotAddVariableComponent
      */
     private applyMask(variable: string): string {
-        return `$!{${variable}}`;
+        return `$dotContentMap.${variable}`;
```

Comment:
```suggestion
        return '${dotContentMap' + variable + '}'
```

As per @wezell the result should be `${dotContentMap.variable}`

Reply:
Noted